### ISSUE TITLE
DescribeInstanceStatus returns empty InstanceStatuses list.

### ISSUE THREAD

**Issue by abhi11** [AdRoll/goamz#420]
When passed instanceIds do DescribeInstanceStatus(), I receive a response containing the RequestId, but I am not receiving any data in InstanceStatuses list, even though I have an instance in stop state.
I can see the instance in my console, but the status is not reported when I call  DescribeInstanceStatus()

**Comment by emcenrue** [comment]
same, except my problem is with terminated state as opposed to the stop state

**Comment by abhi11** [comment]
@emcenrue Yes, even for terminate it doesn't work. I had forked the repo and fixed it, if you're only using EC2 and don't need the other packages try this repo https://github.com/betacraft/goamz has same signatures.
